### ISSUE TITLE
Дублирование доменов

### ISSUE THREAD

**Issue by shusharkin** [itdoginfo/allow-domains#45]
Мне кажется, имеет место дублирование украинских доменов в русских списках, т.к. в самом начале идет домен *.ua, а затем уже конкретные сайты, например, *atn.ua, *hromadske.ua, *hyperhost.ua и т.д. Поиском легко увидеть.

**Comment by itdoginfo** [comment]
Да, есть дубляжи. Жить не мешают, скорее для красоты надо причесать будет. Займусь в будущем

**Comment by itdoginfo** [comment]
Done
